client/blocks/plan-storage/index.jsx

```diff
@@ -143,6 +146,9 @@ export function PlanStorage( {
 					onMouseLeave={ hideTooltip }
 					onFocus={ showTooltip }
 					onBlur={ hideTooltip }
+					onClick={ () => {
+						dispatch( recordTracksEvent( 'calypso_hosting_overview_need_more_storage_click' ) );
```

Comment:
This tracks event was previously fired when the nested link was clicked. The event isn't validated, so there's no description or other data for this new usage to contradict, and it isn't part of any funnels. With that in mind, I've carried the name over rather than creating a new event, as it's usage is very close to the original implementation.

Reply:
I think that's fair 👍